semstr/convert.py

```diff
@@ -171,6 +171,34 @@ def to_sdp(passage, test=False, tree=False, mark_aux=False, **kwargs):
     return SdpConverter(mark_aux=mark_aux, tree=tree).to_format(passage, test=test, format=kwargs.get("format"))
 
 
+def from_ptb(lines, passage_id=None, return_original=False, **kwargs):
+#def from_ptb(passage, passage_id=None, return_original=False, **kwargs):
+    """Converts from parsed text in Penn Treebank mrg format to a Passage object.
+
+    :param lines: iterable of lines in Penn Treebank mrg format, describing a single passage.
+    :param passage_id: ID to set for passage, overriding the ID from the file
+    :param return_original: return triple of (UCCA passage, Mrg string, sentence ID)
+
+    :return generator of Passage objects
+    """
+    from semstr.conversion.ptb import PtbConverter
+    return PtbConverter().from_format(lines, passage_id=passage_id, return_original=return_original,
+                                      format=kwargs.get("format"))
+
+
+def to_ptb(passage, test=False, tree=False, **kwargs):
+    """ Convert from a Passage object to a string in Penn TreeBank mrg format (export)
```

Comment:
It's not `export`, that's the name of another format.

Reply:
My bad, forgot to change it after copying it. I removed it.